services/core/measurements/transport/mq.go

```diff
@@ -62,7 +76,7 @@ func StartMQ(svc *measurements.Service, conn *mq.AMQPConnection, queue, xchg, er
 				}
 				msg.Ack(false)
 			case <-done:
-				break
+				return
```

Comment:
is this change in behaviour intended? makes sense you want to return out of the function when done instead of breaking and continuing in the next iteration. just making sure

Reply:
Yeah, the linter picked it up. Break does nothing there as the loop just continues. When done is closed the transport should be exited as well.

Reply:
check, sounds good